src/components/CardsList.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Stack } from '@mui/material'
+import { Card } from './Card.tsx'
+import { InfoCard } from '../types'
+import { CustomisedTypography } from './CustomisedTypography'
+import CustomisedButton from './CustomisedButton.tsx'
+
+interface CardsListProps {
+  cards: Array<InfoCard>
+}
+
+export function CardsList({ cards }: CardsListProps) {
+  return (
+    <Stack alignItems="center">
+      <CustomisedTypography fontSize="40px" marginTop="140px" mb="50px">
+        Working with GET request
+      </CustomisedTypography>
+      <Box
+        display="flex"
+        flexWrap="wrap"
+        gap="29px"
+        justifyContent="center"
+        mb="50px"
+      >
+        {cards.map((item, index) => (
+          <Card
+            key={index}
```

Comment:
Awful practice
We need to use uuid for prevent react wrong indexing

Reply:
fixed
`{cards.map(({ title, description, email, phone }) => (
          <Card
            key={uuidv4()}
            title={title}
            description={description}
            email={email}
            phone={phone}
          />
        ))}`